use-timescale/data-tiering/enabling-data-tiering.md

```diff
@@ -1,46 +1,222 @@
-1.  In the `Overview` tab, locate the `Tiered Storage` card, and click
-    `Enable tiered storage`. Confirm the action.
-1.  Tiered storage can take a few seconds to turn on and once activated shows the amount of
-    data that has been tiered. Once enabled, data can be tiered by manually tiering 
-    a chunk or by creating a tiering policy.     
+1.  Run `untier_chunk`:
 
-    <img class="main-content__illustration"
-    src="https://assets.timescale.com/docs/images/enable-data-tiering-ga.png"
-    width={1375} height={944}
-    alt="The Timescale Console showing tiered storage enabled" />
+    ```sql
+    CALL untier_chunk('_hyper_1_1_chunk');
+    ```
+
+1.  See the details of the chunk with the
+    `timescaledb_information.chunks` function:
+
+    ```sql
+    SELECT * FROM timescaledb_information.chunks;
+    ```
+
+    The output looks something like this:
+
+    ```sql
+    -[ RECORD 1 ]----------+-------------------------
+    hypertable_schema      | public
+    hypertable_name        | sample
+    chunk_schema           | _timescaledb_internal
+    chunk_name             | _hyper_1_4_chunk
+    primary_dimension      | ts
+    primary_dimension_type | timestamp with time zone
+    range_start            | 2023-02-16 00:00:00+00
+    range_end              | 2020-03-23 00:00:00+00
+    range_start_integer    |
+    range_end_integer      |
+    is_compressed          | f
+    chunk_tablespace       |
+    data_nodes             |
+    ```
 
 </Procedure>
 
-After tiered storage is enabled you must either [manually tier data][manual-tier-chunk] or [setup a tiering policy][creating-data-tiering-policy] 
-to begin tiering data from your hypertables.
 
+## Disable tiering 
+
+If you no longer want to use tiered storage for a particular hypertable, you
+can drop the associated metadata by calling the `disable_tiering` function.
+
+<Procedure>
+
+1. Call `remove_tiering_policy` and drop any tiering policy associated with this hypertable.
+
+1. Make sure that there is no tiered data associated with this hypertable:
+
+    1. List the tiered chunks associated with this hypertable:
+   
+       ```sql
+       select * from timescaledb_osm.tiered_chunks 
+       ```
+
+    1. If you have any tiered chunks, either untier this data, or drop these chunks from tiered storage.
+
+1. Use `disable_tiering` to drop all tiering-related metadata for the hypertable:
+
+   ```sql
+   select disable_tiering('my_hypertable_name');
+   ```
+
+1. Verify that tiering has been disabled by listing the hypertables that have tiering enabled:
+
+   ```sql
+   select * from timescaledb_osm.tiered_hypertables;
+   ```
+
+</Procedure>
 
-[manual-tier-chunk]: /use-timescale/:currentVersion:/data-tiering/manual-tier-chunk/
-[creating-data-tiering-policy]: /use-timescale/:currentVersion:/data-tiering/creating-data-tiering-policy/
+[data-retention]: /use-timescale/:currentVersion:/data-retention/
+[console]: https://console.cloud.timescale.com/
```

Comment:
Can you update to https://console.cloud.timescale.com/dashboard/services so it takes the user directly to the services page please.